wordpress-mobile/WordPress-Android
# Fix Classic Block in the Unsupported Block editor

This PR introduces a fix for the block editor for better support of the Classic Block.

Gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2550

It's based on gb-mobile v1.33.1.

To test:

1. Switch to html mode and write some text in-between two blocks, making sure the text is not inside the block delimiters
2. Switch back to visual mode
3. There should be an unsupported block displayed between the blocks the text was inserted
4. Use the "?" button of the unsupported block and then "Edit block in web browser" to launch the Unsupported Block editor
5. Notice the text being displayed inside a "Classic" block without an error 🎉
6. Make some changes to the text and hit the "✔️" icon in the upper right to commit the changes
7. Switch to html mode again and notice that the text includes the changes made inside the Unsupported Block editor 🎉

PR submission checklist:
